### ISSUE TITLE
Flex items shown compressed when page wrapping is set.

### ISSUE THREAD

**Issue by paulewetzel** [diegomura/react-pdf#1114]
**Describe the bug**
I created a table with flexbox which happened to start near the first page break of the document. If I set wrap={false} on the flexboxed View, everything renders correctly, after the table is pushed to the next page. Unfortunately, when the table contains more items than can fit on one page, I get a memory error, the browser hangs and must be restarted. On the other hand, when I set it back to wrap={true},  the records that appear before the page break are all compressed together, as if the styles were not applied.

**Expected behavior**
I expect the table flexbox rows to look the same, even when the page is split.

**Screenshots**
When there is a page break and wrap is set to "false"
![Screen Shot 2021-02-10 at 7 33 18 PM](https://user-images.githubusercontent.com/896811/107604024-8cdd6580-6be3-11eb-9f78-60426cb88e96.png)

When there is not a page break and wrap is set to "true"
![Screen Shot 2021-02-10 at 7 30 09 PM](https://user-images.githubusercontent.com/896811/107604025-8e0e9280-6be3-11eb-8ee5-96e0e7ec5959.png)

**Desktop (please complete the following information):**
 I am on a mac, but am working through a Win10 Pro virtualized desktop as required by my client.
 React version 17.0.1

**Comment by diegomura** [comment]
Can you provide a valid snippet to replicate it? Thanks!

**Comment by paulewetzel** [comment]
Yes, I am kind of under the gun right now at the end of a project, but as soon as I get some time I will try to work up an example.

**Comment by paulewetzel** [comment]
Sorry, I will be getting back to this item in the next few weeks when I do the tech-debt cleanup and circle back to this issue. I have not forgotten.

**Comment by paulewetzel** [comment]
Good news. I finally worked on this item today and was able to get the resultant PDF in a state that no longer produced a collapsing layout. 

While poking around, I noticed that right before I encountered this bug, someone had changed a function that I was calling to conditionally generate image URLs. Once I checked out the previous version of that module, the collapsing flexbox issue ceased. 

Since I can no longer produce this error, I am closing this item. Thank you for your patience and sorry for the false alarm.
